docs/src/components/ComponentPage/AndroidComponentPage.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import BpkLink from 'bpk-component-link';
+
+import ComponentScreenshots from '../DocsPageBuilder/ComponentScreenshots';
+
+import ComponentPage from './ComponentPage';
+
+type Props = {
+  documentationId: string,
+  readme: string,
+  screenshots: [
+    {
+      id: string,
+      screenshots: {
+        altText: string,
+        height: number,
+        src: string,
+        subText: string,
+        title: string,
+        width: number,
+      },
+      title: string,
+    },
+  ],
+  usageTable: ?{
+    dos: Array<string>,
+    donts: Array<string>,
+  },
+};
+
+const AndroidComponentPage = (props: Props) => {
+  const { documentationId, screenshots, readme, usageTable } = props;
+  const screenshotsAsExamples = screenshots.map(screenshot => ({
+    id: screenshot.id,
+    title: screenshot.title,
+    content: <ComponentScreenshots screenshots={screenshot.screenshots} />,
+  }));
+  return (
+    <ComponentPage
+      usageTable={usageTable}
+      examples={screenshotsAsExamples}
+      readme={readme}
+      additionalContent={[
+        {
+          id: 'docs',
+          title: 'Class reference',
+          content: (
+            <BpkLink href={`/android/versions/latest/${documentationId}`} blank>
```

Comment:
Are we concerned that if the documentationId does not exist for some reason it would link to an empty page or not found?

Reply:
Right now I'm operating under the assumption that every Android page has an accompanying docs page link.

I think it's preferable to keep using that assumption until it's proven wrong (we'd discover the error very quickly when we try to roll this out to all pages) than to add a null check that might not be needed.

Reply:
I guess but its better to be preventative and it not be an issue than, let it roll out and give our consumers a bad experience, then have to do a fix rather than have something in there that would prevent issues and broken functionality in the first place.

Plus we are already doing it for the Web page so it would kind of make sense to do it here?

Reply:
Gonna resolve this conversation, as the same discussion applies to both Android and iOS, so we can just centralise it on the iOS thread